static/js/message_list_view.js

```diff
@@ -439,7 +439,7 @@ MessageListView.prototype = {
                     var text = $(this).attr("title");
                     return ":" + text + ":";
                 });
-             }
+            }
```

Comment:
Better to have this in a different commit, also maybe investigate why the linter didn't catch this?